### PR TITLE
GSoD 2021 Final Blog Post Landing Page Team

### PR DESCRIPTION
**Pull request by sajen-k**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Review by iamrajiv — APPROVED** [review]

File: jekyll/_posts/2021-11-16-wechaty-landing-page-final-blog.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+4)Talks Section of the new Wechaty landing page (Section 4) [#1267](https://github.com/wechaty/wechaty.js.org/pull/1267)\
+5)Honors and Awards section of the new wechaty landing page (Section 5) [#1269](https://github.com/wechaty/wechaty.js.org/pull/1269)\
+6)voice of developers section of the new wechaty landing page (Section 6) [#1271](https://github.com/wechaty/wechaty.js.org/pull/1271)\
+7)Sponsors section of the new wecahty landing page (Section 7) [#1272](https://github.com/wechaty/wechaty.js.org/pull/1272)\
+8)Footer section of the new wecahty landing page (Section 8) [#1292](https://github.com/wechaty/wechaty.js.org/pull/1292)\
+9)Dark Theme Implementation in the new landing page [#1366](https://github.com/wechaty/wechaty.js.org/pull/1366)\
+10)link colour according to the theme [#1365](https://github.com/wechaty/wechaty.js.org/pull/1365)\
+11)Bugs and misalignments rectification [#1360](https://github.com/wechaty/wechaty.js.org/pull/1360)\
+12)Several alignment issues resolved [#1358](https://github.com/wechaty/wechaty.js.org/pull/1358)\
+13)Error sections fixed in the new landing page [#1357](https://github.com/wechaty/wechaty.js.org/pull/1357)
```
Comment (Rohitesh-Kumar-Jain) [inline jekyll/_posts/2021-11-16-wechaty-landing-page-final-blog.md:92]: pls embed the introduction page video as well as it has competed, right?

**Review by shraddhavp — APPROVED** [review]
LGTM just turn the CLI green

File: jekyll/_posts/2021-11-16-wechaty-landing-page-final-blog.md (hunk 1 above)
Comment (sajen-k) [reply jekyll/_posts/2021-11-16-wechaty-landing-page-final-blog.md:92]: That introduction video is a draft, and it has not been authorized by the mentors or other community members, and the ADVIDS team is seeking input on it, so the video is not finished.

**Review by arnabiscreating — APPROVED** [review]
Looks good

**Pull request merged**